src/firejail/checkcfg.c

```diff
@@ -424,6 +424,12 @@ static const char *const compiletime_support =
 		"enabled"
 #else
 		"disabled"
+#endif
+	"\n\t- wayland security context support is "
+#ifdef HAVE_WAYLAND_SECURITY_CONTEXT
+		"enabled"
+#else
+		"disabled"
```

Comment:
By default put the wayland code right before the x11 code so that it's more or
less sorted.

This applies to all files (except env.c, which is already sorted).